src/assets/scripts/client/constants/canvasConstants.js

```diff
@@ -14,3 +14,21 @@ export const DEFAULT_CANVAS_SIZE = {
     HEIGHT: 480,
     WIDTH: 640
 };
+
+/**
+ * Various options for drawing aircraft on canvas
+ *
+ * @enum AIRCRAFT_DRAW_OPTIONS
+ * @type {Object}
+ * @final
+ */
+export const AIRCRAFT_DRAW_OPTIONS = {
+    /**
+     * Length of PTL lines, in minutes. Standard is 1.
```

Comment:
honestly I have no idea actually. it's the name of a big green menu bar across the top that you use to change settings of the scope.